pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.1</version>
+    <version>3.42</version>
```

Comment:
could be `3.43` now. If I recall correct, the next version has a `javadoc` fix as well, no?